src/main/java/com/lorepo/icplayer/client/module/text/GapInfo.java

```diff
@@ -150,18 +150,23 @@ public void setPlaceHolder(String placeHolder) {
 	public String getPlaceHolder() {
 		return placeHolder;
 	}
-	
-	public int getLongestAnswerLength() {
-		int longestAnswer = 0;
+
+	public String getLongestAnswer() {
+		int longestAnswerLength = 0;
+		String longestAnswer = "";
 		
 		for (String answer : answers) {
-			if (longestAnswer < answer.length()) {
-				longestAnswer = answer.length();
+			if (longestAnswerLength < answer.length()) {
```

Comment:
to zawsze będzie true, bo nie aktualizujesz longestAnswerLength, tylko longestAnswer.